.github/workflows/R-CMD-check.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: any::rcmdcheck
+          extra-packages: any::rcmdcheck, patchwork@1.3.1
```

Comment:
Unrelated to the actual matter at hand, but necessary to pass R CMD check on Windows on oldrel-4.